### ISSUE TITLE
Use MediaType, HttpStatus and HttpHeaders instead of strings in our code base

### ISSUE THREAD

**Issue by steve-todorov** [strongbox/strongbox#1497]
# Task Description

There are some left-overs in our code base where we are still using strings instead of the appropriate enums when setting headers and their values. You need to find all such classes/test cases and convert them to use the appropriate enums from: 

```
import org.springframework.http.HttpHeaders;
import org.springframework.http.HttpStatus;
import org.springframework.http.MediaType;
```

Example:

1. If you happen to find test cases which have the following code:
    ```
    RestAssuredMockMvc.given()
                      .contentType("application/json")
                      .header("Accept", "application/json")
    ```
    You should convert it to this:
    ```
    import io.restassured.module.mockmvc.RestAssuredMockMvc.given;

    given().contentType(MediaType.APPLICATION_JSON_VALUE)
           .accept(MediaType.APPLICATION_JSON_VALUE)
    ```

2. If you notice this code:
    ```
    given().header(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE)
    ```
    Replace with:
    ```
    given().accept(MediaType.APPLICATION_JSON_VALUE)
    ```
    Both are doing the same thing, but the replacement is easier to read :)

3. If in doubt - join the channel and ask or write a message here :)

# Task Relationships

This task:
* Relates to: #1477 (comments section)

# Help

* [Our chat](https://chat.carlspring.org/)

**Comment by cagans42** [comment]
Can I take this up ? @steve-todorov

**Comment by steve-todorov** [comment]
Hi @cagans42 - sure thing! 

Feel free to [join our chat](https://chat.carlspring.org) if you have questions! :)

**Comment by anki2189** [comment]
@cagans42 : Have you picked this task ? If not can I work on it.?

**Comment by carlspring** [comment]
@anki2189 , @harishmaA,

@ze12augusto has already submitted a pull request for this and it's already been approved (with a couple of minor change requests).

Please, join our [chat channel](https://chat.carlspring.org/), if you'd like to find out what else you can help with.

**Comment by carlspring** [comment]
Closed via #1498

**Comment by HarishmaA** [comment]
Sure..Thanks
